packages/@adobe/spectrum-css-temp/components/textfield/index.css

```diff
@@ -263,7 +263,7 @@ governing permissions and limitations under the License.
 }
 
 /* styles the textfield properly if the left icon is provided */
-.spectrum-Textfield-inputIcon {
+.spectrum-Textfield-input.spectrum-Textfield-inputIcon {
```

Comment:
Specificity has to win over `.spectrum-Textfield-multiLine .spectrum-Textfield-input`, or the `padding` shorthand there will override the `padding-inline-start` here. Didn't see this before because of the extra specificity of the old logical properties compilation.